app/services/conference-service.js

```diff
@@ -149,7 +149,25 @@ import 'angular-cache'
               )
             }
 
+            function getAgendas(ids){
+              var oidArray=[];
+
+              for (var i=0; i<ids.length; i++) {
+                oidArray.push({
+                    '$oid': ids[i]
+                });
+              }
```

Comment:
you can now use new syntax in www 
```suggestion
              const oidArray= ids.map(id=>{'$oid': id});
```

Reply:
Unless I am wrong I though rollup babeled it down now.